editor/src/clj/editor/model_scene.clj

```diff
@@ -400,14 +433,15 @@
                                     :icon mesh-icon
                                     :view-types [:scene :text]))
 
-(defn- update-vb [^GL2 gl ^VertexBuffer vb data]
-  (let [{:keys [mesh world-transform vertex-space vertex-attribute-bytes]} data
-        world-transform (doto (Matrix4d.) (math/clj->vecmath world-transform))]
-    (mesh->vb! vb world-transform vertex-space vertex-attribute-bytes mesh)))
+(defn- update-vb [^GL2 _gl ^VertexBuffer vb data]
+  (let [{:keys [mesh-renderable-data ^Matrix4d world-transform ^Matrix4d normal-transform vertex-attribute-bytes]} data]
+    (mesh->vb! vb world-transform normal-transform vertex-attribute-bytes mesh-renderable-data)
```

Comment:
Added `normal-transform`, changed final argument to `renderable-data` for the mesh.